dask_planner/src/parser.rs

```diff
@@ -375,4 +437,48 @@ impl<'a> DaskParser<'a> {
             schema_name,
         })))
     }
+
+    /// Retrieves the table_schema and table_name from a `TableFactor` instance
+    fn elements_from_tablefactor(tbl_factor: &TableFactor) -> (String, String) {
```

Comment:
Would it make sense to break this function (along with`options_from_tablefactor`)  off into a separate module? It seems like a utility for `parse_create_table` that contrasts with the other `parse_*` methods of this module.